src/Umbraco.Infrastructure/Persistence/NPocoSqlExtensions.cs

```diff
@@ -88,6 +88,17 @@ public static Sql<ISqlContext> WhereLike<TDto>(this Sql<ISqlContext> sql, Expres
             return sql;
         }
 
+        public static Sql<ISqlContext> Union(this Sql<ISqlContext> sql, Sql<ISqlContext> sql2)
+        {
+            return sql.Append( " UNION ").Append(sql2);
+        }
+        public static Sql<ISqlContext>.SqlJoinClause<ISqlContext> InnerJoinNested(this Sql<ISqlContext> sql, Sql<ISqlContext> nestedQuery, string alias)
+        {
+            return new Sql<ISqlContext>.SqlJoinClause<ISqlContext>(sql.Append("INNER JOIN (").Append(nestedQuery)
+                .Append($") [{alias}]"));
+
+        }
```

Comment:
```suggestion
        public static Sql<ISqlContext> Union(this Sql<ISqlContext> sql, Sql<ISqlContext> sql2)
        {
            return sql.Append( " UNION ").Append(sql2);
        }
        
        public static Sql<ISqlContext>.SqlJoinClause<ISqlContext> InnerJoinNested(this Sql<ISqlContext> sql, Sql<ISqlContext> nestedQuery, string alias)
        {
            return new Sql<ISqlContext>.SqlJoinClause<ISqlContext>(sql.Append("INNER JOIN (").Append(nestedQuery)
                .Append($") [{alias}]"));
        }
```
Small formatting change 👍